config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -939,6 +940,7 @@ presubmits:
         - --cluster=
         - --extract=local
         - --flush-mem-after-build=true
+        - --gcp-node-size=e2-medium
```

Comment:
Sorry - missed that. I don't think we should be changing presubmits especially for release branches in the first step...